[BE] delete frontend.cpp

wanchaol opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #66649
* __->__ #66581

c10d/frontend.cpp was originally proposed to introduce pure C++ API and use TorcBind to share python level API with TorchScript. This is no longer needed, so delete this to reduce code redundancy.

Differential Revision: [D31627107](https://our.internmc.facebook.com/intern/diff/D31627107/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D31627107/)!

cc @pietern @mrshenli @pritamdamania87 @zhaojuanmao @satgera @rohan-varma @gqchen @aazzolini @osalpekar @jiayisuse @SciPioneer @H-Huang

Commit:
[BE] delete frontend.cpp

c10d/frontend.cpp was originally proposed to introduce pure C++ API and use TorcBind to share python level API with TorchScript. This is no longer needed, so delete this to reduce code redundancy.

Differential Revision: [D31627107](https://our.internmc.facebook.com/intern/diff/D31627107/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D31627107/)!

[ghstack-poisoned]

Commit:
Update on "[BE] delete frontend.cpp"


c10d/frontend.cpp was originally proposed to introduce pure C++ API and use TorcBind to share python level API with TorchScript. This is no longer needed, so delete this to reduce code redundancy.

Differential Revision: [D31627107](https://our.internmc.facebook.com/intern/diff/D31627107/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D31627107/)!

[ghstack-poisoned]

Commit:
Update on "[BE] delete frontend.cpp"


c10d/frontend.cpp was originally proposed to introduce pure C++ API and use TorcBind to share python level API with TorchScript. This is no longer needed, so delete this to reduce code redundancy.

Differential Revision: [D31627107](https://our.internmc.facebook.com/intern/diff/D31627107/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D31627107/)!

[ghstack-poisoned]

Commit:
Update on "[BE] delete frontend.cpp"


c10d/frontend.cpp was originally proposed to introduce pure C++ API and use TorcBind to share python level API with TorchScript. This is no longer needed, so delete this to reduce code redundancy.

Differential Revision: [D31627107](https://our.internmc.facebook.com/intern/diff/D31627107/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D31627107/)!

[ghstack-poisoned]

Commit:
Update on "[BE] delete frontend.cpp"


c10d/frontend.cpp was originally proposed to introduce pure C++ API and use TorcBind to share python level API with TorchScript. This is no longer needed, so delete this to reduce code redundancy.

Differential Revision: [D31627107](https://our.internmc.facebook.com/intern/diff/D31627107/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D31627107/)!

[ghstack-poisoned]

Commit:
Update on "[BE] delete frontend.cpp"


c10d/frontend.cpp was originally proposed to introduce pure C++ API and use TorcBind to share python level API with TorchScript. This is no longer needed, so delete this to reduce code redundancy.

Differential Revision: [D31627107](https://our.internmc.facebook.com/intern/diff/D31627107/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D31627107/)!

[ghstack-poisoned]

rohan-varma reviewed (approved):
LGTM although please address @cbalioglu 's comments

Commit:
Update on "[BE] delete frontend.cpp"


c10d/frontend.cpp was originally proposed to introduce pure C++ API and use TorcBind to share python level API with TorchScript. This is no longer needed, so delete this to reduce code redundancy.

Differential Revision: [D31627107](https://our.internmc.facebook.com/intern/diff/D31627107/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D31627107/)!

[ghstack-poisoned]

Commit:
Update on "[BE] delete frontend.cpp"


c10d/frontend.cpp was originally proposed to introduce pure C++ API and use TorcBind to share python level API with TorchScript. This is no longer needed, so delete this to reduce code redundancy.

Differential Revision: [D31627107](https://our.internmc.facebook.com/intern/diff/D31627107/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D31627107/)!

[ghstack-poisoned]

Commit:
Update on "[BE] delete frontend.cpp"


c10d/frontend.cpp was originally proposed to introduce pure C++ API and use TorcBind to share python level API with TorchScript. This is no longer needed, so delete this to reduce code redundancy.

Differential Revision: [D31627107](https://our.internmc.facebook.com/intern/diff/D31627107/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D31627107/)!

[ghstack-poisoned]

Commit:
Update on "[BE] delete frontend.cpp"


c10d/frontend.cpp was originally proposed to introduce pure C++ API and use TorcBind to share python level API with TorchScript. This is no longer needed, so delete this to reduce code redundancy.

Differential Revision: [D31627107](https://our.internmc.facebook.com/intern/diff/D31627107/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D31627107/)!

[ghstack-poisoned]
